src/Oro/Bundle/SecurityBundle/Authentication/Token/OrganizationContextTokenTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Oro\Bundle\SecurityBundle\Authentication\Token;
+
+use Oro\Bundle\OrganizationBundle\Entity\Organization;
+
+trait OrganizationContextTokenTrait
+{
+    /** @var  Organization */
+    protected $organization;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getOrganizationContext()
+    {
+        return $this->organization;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setOrganizationContext(Organization $organization)
```

Comment:
Please add comments what this function do. {@inheritdoc} here is not good here because you do not extend any class, etc. For example, you can see https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/ActivityBundle/Model/ExtendActivity.php